_episodes/04-file-io.md

```diff
@@ -8,8 +8,8 @@ objectives:
 - "Copy files to and from the docker container"
 - "Mount directories to be accessed and manipulated by the docker container"
 keypoints:
-- "We can copy files with ``docker cp``"
-- "We can mount volumes with ``docker run -v <path on host>:<path in container> <image>``"
+- "Copy files with ``docker cp``"
```

Comment:
These should be single backticks though it'll probably render OK

Reply:
Done